### DISCUSSION TITLE
templates

### DISCUSSION THREAD

**Discussion by priyanshuparashar** [adamspd/django-appointment#202] [Q&A]
I am not able to config the templates. The default django admin view is there nothing like the one shown in docs or readme file 
If there is any tutorial please share. I have made the following changes in settings.py :
TEMPLATES = [
    {
        "BACKEND": "django.template.backends.django.DjangoTemplates",
        "DIRS": [os.path.join(BASE_DIR, 'templates')],
        "APP_DIRS": True,
        "OPTIONS": {
            "context_processors": [
                "django.template.context_processors.debug",
                "django.template.context_processors.request",
                "django.contrib.auth.context_processors.auth",
                "django.contrib.messages.context_processors.messages",
            ],
        },
    },
]
APPOINTMENT_BASE_TEMPLATE = 'base.html'
# (optional) Specify a different base template for the admin panel
APPOINTMENT_ADMIN_BASE_TEMPLATE = 'base.html'

**Comment by adamspd** [comment]
> I am not able to config the templates. The default django admin view is there nothing like the one shown in docs or readme file 
> 
> If there is any tutorial please share. I have made the following changes in settings.py :
> 
> TEMPLATES = [
> 
>     {
> 
>         "BACKEND": "django.template.backends.django.DjangoTemplates",
> 
>         "DIRS": [os.path.join(BASE_DIR, 'templates')],
> 
>         "APP_DIRS": True,
> 
>         "OPTIONS": {
> 
>             "context_processors": [
> 
>                 "django.template.context_processors.debug",
> 
>                 "django.template.context_processors.request",
> 
>                 "django.contrib.auth.context_processors.auth",
> 
>                 "django.contrib.messages.context_processors.messages",
> 
>             ],
> 
>         },
> 
>     },
> 
> ]
> 
> APPOINTMENT_BASE_TEMPLATE = 'base.html'
> 
> # (optional) Specify a different base template for the admin panel
> 
> APPOINTMENT_ADMIN_BASE_TEMPLATE = 'base.html'
> 
> 

What are the errors you're having ?
